nutils/function.py

```diff
@@ -494,9 +410,18 @@ class Custom(Array):
   pointwise axes and the shape of the result passed to :class:`Custom` should
   not include the pointwise axes.
 
+  For internal reasons, both ``evalf`` and ``partial_derivative`` must be
+  decorated as `classmethod` or `staticmethod`, meaning that they will not
```

Comment:
The single backticks should be double backticks.

Reply:
fixed